### ISSUE TITLE
Accessibility: Make a version of the WinUI Gallery that meets Accessibility requirements.

### ISSUE THREAD

**Issue by tinodo** [microsoft/WinUI-Gallery#1358]
### Issue type

enhancement

### Which version of the app?

WinUI 3 Gallery

### Description

The current version of the WinUI Gallery doesn't meet Accessibility requirements. Most enterprises will require applications to comply with the accessibility requirements stated here: [https://www.w3.org/WAI/](https://www.w3.org/WAI/)

It would be great to have either an option or a version of the app that meets these requirements, and shows how WinUI applications could meet these requirements.

This applies to borders (Luminosity ratio) , visual focus indicators (only at the bottom), and many more....

### Windows version

Windows 11 22H2 (22621)

### Additional context

Our applications have to meet these requirements.

**Comment by chingucoding** [comment]
I assume, this issue is because you found issues with the WinUI 3 Gallery, is this correct? There also is #1361 is also touching on this as this would scan every page of this app to pass an Axe scan.

**Comment by karkarl** [comment]
We are working on it and addressing specific issues. It is our intent to keep working towards this goal. As @chingucoding mentioned, we have recently implemented Axe scans on every page, and other fixes are underway.
